tests/integration/src/test/resources/org/mule/properties/attachment-transformer-test-case.xml

```diff
@@ -95,7 +95,7 @@
         <expression-transformer expression="#[message.outboundAttachments['attach']]"/>
         <expression-transformer expression="#[groovy:message.getPayload().getInputStream()]"/>
         <object-to-string-transformer/>
-        <test:assert expression="#[groovy:message.getPayloadAsString().equals('some text')]"/>
```

Comment:
I mean, how can you do the same using groovy. In this case, instead of returning the value (which is a String) it could be good to show how to use the transformation service